moveit_demo_nodes/run_move_group/config/panda_controllers.yaml

```diff
@@ -1,3 +1,16 @@
+controller_manager:
+  ros__parameters:
+    update_rate: 2  # Hz
```

Comment:
Isn't that a little slow?

Reply:
this is probably copy-paste from the examples :D